python/lsst/ts/ataos/model.py

```diff
@@ -53,6 +68,10 @@ def __init__(self, log):
         self.poly_v = np.poly1d(self.config["hexapod_v"])
         self.poly_chromatic = np.poly1d(self.config["chromatic_dependence"])
 
+        self.m1_lut_elevation_limits = [0.0, 90.0]
```

Comment:
same deal, this is in the config, same with the lines below

Reply:
Same argument... :)